frontend/src/hooks/usePersonnel.tsx

```diff
@@ -16,8 +21,17 @@ const usePersonnel = ({
     })();
   }, [personnelId]);
 
+  const getAvailability = async (from: string, to: string) => {
+    const response = await AxiosPrivate.get(
```

Comment:
Can we move this to the endpoint utils? (I just like having the list of all of the endpoints that we are using in one file - totally just a preference thing though if you feel otherwise :) )

Reply:
Let's tag this on as tech debt, I think `useTable` has its own requests too.